File: Guanabara - POO Java/Meus exercícios POO/Aula06/src/main/java/aula06/Controlador.java
package aula06;
public interface Controlador {
    public abstract void ligar();//Declarar um método como abstract quer dizer que o código não será definido no local, mas em outro local/arquivo.
    public abstract void desligar();
    public abstract void abrirMenu();
    public abstract void fecharMenu();
    public abstract void maisVolume();
    public abstract void menosVolume();
    public abstract void ligarMudo();
    public abstract void desligarMudo();
    public abstract void play();
    public abstract void pause();
}

File: PHP/exercicios-PHPOO/Aula06/Controlador.php
<?php
interface Controlador {
    public  function ligar();
    public  function desligar();
    public  function abrirMenu();
    public  function fecharMenu();
    public  function maisVolume();
    public  function menosVolume();
    public  function ligarMudo();
    public  function desligarMudo();
    public  function play();
    public  function pause();
}
//não é necessário explicitar o método como abstract ao declará-lo em um interface